src/python/pants/backend/project_info/paths.py

```diff
@@ -90,35 +93,36 @@ def find_paths_breadth_first(
 
 
 @goal_rule
-async def paths(
-    console: Console, addresses: Addresses, paths_subsystem: PathsSubsystem
-) -> PathsGoal:
+async def paths(console: Console, paths_subsystem: PathsSubsystem) -> PathsGoal:
```

Comment:
We weren't using `addresses`, and still aren't, so can remove it from the signature.